source/patterns/@aws-solutions-constructs/core/test/elasticsearch-helper.test.ts

```diff
@@ -132,8 +149,154 @@ test('Test override SnapshotOptions for buildElasticSearch', () => {
   });
 });
 
-test('Test override ES version for buildElasticSearch', () => {
+test('Test VPC with 1 AZ, Zone Awareness Disabled', () => {
   const stack = new Stack();
+
+  const vpc = deployVpc(stack);
```

Comment:
we already have defaults.getTestVpc() - do we really need another function that creates VPCs for tests? Can we add optional paramter(s) to getTestVpc() to handle any requirement you have that it doesn't yet adddress?

Reply:
Did not see the original getTestVpc() function. I added the optional parameter and replaced it in the new code.